apps/web/src/components/GetStarted/ResourceGrid/ResourceCard.tsx

```diff
@@ -0,0 +1,49 @@
+'use client';
+
+import { useCallback } from 'react';
+import Link from 'next/link';
+import classNames from 'classnames';
+import { ActionType, ComponentType } from 'base-ui/utils/logEvent';
+import { useAnalytics } from '../../../../contexts/Analytics';
+import { ResourceCardProps } from '../resourceTypes';
+
+export default function ResourceCard({
+  href,
+  title,
+  description,
+  topLeft = <span />,
+  topRight = <span />,
+  colorOne,
+  colorTwo,
+}: ResourceCardProps) {
+  const { logEventWithContext } = useAnalytics();
+
+  const resourceCardWrapperStyle = classNames(
+    'group border-2 p-7 text-white',
+    `odd:bg-${colorOne} odd:border-${colorOne}`,
+    `even:bg-${colorTwo ?? colorOne} even:border-${colorTwo ?? colorOne}`,
+    'duration-200',
+    'hover:scale-105 hover:shadow-lg hover:shadow-gray-80',
+  );
+
+  const handleClick = useCallback(() => {
+    logEventWithContext(title.replace(/ /g, '_').replace(/\W/g, ''), ActionType.click, {
+      componentType: ComponentType.link,
+    });
+  }, [logEventWithContext, title]);
+
+  return (
+    <div key={href} className={resourceCardWrapperStyle}>
+      <Link href={href} onClick={handleClick}>
+        <div className="flex w-full flex-row justify-between">
+          <div>{topLeft}</div>
+          <div>{topRight}</div>
+        </div>
```

Comment:
when content is null we should not render any dom

Reply:
removed the unnecessary divs